kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyFrontendHandler.java

```diff
@@ -479,7 +484,7 @@ void inConnecting(
         // reads Kafka requests, as the message flows are reversed. This is also the opposite of the order that Filters are declared in the Kroxylicious configuration
         // file. The Netty Channel pipeline documentation provides an illustration https://netty.io/4.0/api/io/netty/channel/ChannelPipeline.html
         if (logFrames) {
-            pipeline.addFirst("frameLogger", new LoggingHandler("io.kroxylicious.proxy.internal.UpstreamFrameLogger", LogLevel.INFO));
+            pipeline.addFirst("frameLogger", new FrameLoggingHandler("io.kroxylicious.proxy.internal.UpstreamFrameLogger-" + sessionId, LogLevel.INFO));
```

Comment:
I think your motativation of adding the sessionId to the catogory is so that any frame logging carries the session.  I'm wondering about the wisdom of the `-` separator.  That's going to mean that anybody trying to control UpstreamFrameLogger from the log4j config file can't.  Would a `.` separate be a better choice?

Or, maybe have own own SessionAwareLoggingHandler which basically does exactly what Netty's LoggingHandler does, but includes the session id in all the log messages.    I'd just copy LoggingHandler rather than subclassing.